kiwi/filesystem/isofs.py

```diff
@@ -15,7 +15,7 @@
 # You should have received a copy of the GNU General Public License
 # along with kiwi.  If not, see <http://www.gnu.org/licenses/>
 #
-import os
+from distutils.spawn import find_executable
```

Comment:
Uhm, don't. :laughing:  Distutils aren't always installed. If you _package_ Kiwi, you might find out that this will fail "module not found".